crates/api_common/src/post.rs

```diff
@@ -70,6 +70,7 @@ pub struct GetPostResponse {
 pub struct GetPosts {
   pub type_: Option<ListingType>,
   pub sort: Option<SortType>,
+  /// DEPRECATED, use page_v2
```

Comment:
We do need to consider how apps can be compatible with both Lemmy 0.18 and 0.19 at the same time. If we completely change the meaning of page parameter here, it means that apps have to somehow support both old and new apis, and check the Lemmy version to send the correct param. So in this case depreciation makes sense to me. However instead of `page_v2` I would use `next_page` so we dont have to make a breaking change to rename the field in the future.

Reply:
I can rename the parameter though I don't really understand what you mean by "so we don't have to make a breaking change to rename the field in the future"?

Reply:
I also like `page_cursor`, then when we remove `page`, we won't have to rename `page_v2` -> `page`, it can just stay as `page_cursor`

Reply:
ah, i understand. i had never intended for this to be renamed later, just keep it as is. But sure I'll change the name to page_cursor.